docs/UserGuide.md

```diff
@@ -38,11 +43,51 @@ It is optimized for fast typists and replicates a Command Line Interface (CLI) s
 
    * **`exit`** : Exits the app.
 
-6. Refer to the [things to note](#3-things-to-note) below for details of each command.
+6. You can refer to the [things to note](#3-things-to-note) below for details of each command.
 
 --------------------------------------------------------------------------------------------------------------------
+## 3. About Our User Guide
 
-## 3. Things to note
+This guide serves to:<br>
+
+1. Educate new users on how to get started with WorkBook!
+2. Explain the purpose and how to use each and every command!
+3. Display a summary of all the commands you can utilize!
+
+
+
+
+
+--------------------------------------------------------------------------------------------------------------------
+## 4. Exploring our User Guide
+
+
+**Information Box**
+<div markdown="block" class="alert alert-info">
+**:information_source: Info:** Provides extra information that is useful
+</div>
+
+**Warning Box**
+<div markdown="block" class="alert alert-warning">
+**:exclamation: Warning: Important messages**
+</div>
+
+**Tip Box**
+<div markdown="block" class="alert alert-success">
+**:bulb: Tip:** Provides pointers to enhance your experience using the application 
+</div>
+
```

Comment:
I would say these fall under things to note as well

Reply:
no